package.json

```
@@ -64,6 +64,7 @@
     "eslint-plugin-inclusive-language": "~2.2.0",
     "eslint-plugin-jsdoc": "~39.3.6",
     "eslint-plugin-mdx": "~2.0.2",
+    "eslint-plugin-n": "^15.3.0",
```

```suggestion
    "eslint-plugin-n": "~15.3.0",
```

Please re-run pnpm install after that to regenerate lock file